examples/vision/bit.py

```diff
@@ -53,7 +56,7 @@
 SEEDS = 42
 
 np.random.seed(SEEDS)
-tf.random.set_seed(SEEDS)
+seed_generator = keras.random.SeedGenerator(1337)
```

Comment:
What is this for?

Reply:
You shouldn't need to include this line at all.

Reply:
Ok I am changing that!

Reply:
> You shouldn't need to include this line at all.

I think this can be used instead of that ```keras.utils.set_random_seed(SEEDS)``` as mentioned in the documentation https://keras.io/api/utils/python_utils/#setrandomseed-function

Reply:
Yes, you can use `set_random_seed` to seed the example.